ui/components/property-input-entity.vue

```diff
@@ -189,10 +195,6 @@ export default {
     hasNumber() {
       return hasNumber(this.selectedUnit, this.enumValues);
     },
-    selectedUnitIsNumber() {
```

Comment:
Why do we not need this anymore?

Reply:
Because I removed the only usage. max-number now also applies for numbers with unit (i.e. percent or also Hertz) – this is no problem as it's only used with default and highlight value so far.